corehq/apps/user_importer/tests/test_importer.py

```diff
@@ -104,7 +106,7 @@ def test_upload_with_missing_user_id(self):
             self.domain.name,
             [self._get_spec(user_id='missing')],
             [],
-            None,
+            self.uploading_user,
```

Comment:
@calellowitz These changes seem like they might interact with https://github.com/dimagi/commcare-hq/pull/28628 since in that test you're creating the `UserUploadRecord` with a fake user.

Reply:
thanks for sharing that @orangejenny 
I updated with master. Even though the new test added by @calellowitz ran successfully and didn't need uploading user to be present, I passed it anyway as a practice since uploading user can actually never be none in real scenario.
https://github.com/dimagi/commcare-hq/pull/28631/commits/32f50ff3ba2b3573ae045c4bf05b4e2d261194be